httpclient/fixtures/slice_transport.go

```diff
@@ -0,0 +1,51 @@
+package fixtures
+
+import (
+	"fmt"
+	"net/http"
+)
+
+var Failures = SliceTransport{
+	{
+		MatchAny:     true,
+		ReuseRequest: true,
+		Status:       418,
+		Response:     `{"error_code": "NONSENSE", "message": "I'm a teapot"}`,
+	},
+}
```

Comment:
What is this? If it is unused, we don't need to expose it; users can define their own failures per unit test.

Reply:
there are ~200 usages of this in TF unit tests